public/app/features/auth-config/AuthProvidersListPage.tsx

```diff
@@ -98,9 +98,9 @@ export const AuthConfigPageUnconnected = ({
             {providerList
               // Temporarily filter out providers that don't have the UI implemented
               .filter(({ provider }) => !['grafana_com'].includes(provider))
-              .map(({ provider, settings }) => (
+              .map(({ provider, settings }, i) => (
```

Comment:
This is only needed if it's possible to have multiple providers with the same name.